proseco/tests/test_guide.py

```diff
@@ -211,8 +211,8 @@ def test_check_mag_spoilers():
 
     # The mag spoiler check only works on stars that are within 10 pixels in row
     # or column, so don't bother simulating stars outside that distance
-    r_dists = np.arange(-9.25, 10, 1)
-    c_dists = np.arange(-9.5, 10, 1)
+    r_dists = np.arange(-9.25, 10, 2)
+    c_dists = np.arange(-9.5, 10, 2)
```

Comment:
That's fine.  The bigger the steps, the less sensitive the test, but if we modify the algorithm we can always decrease the step size again for validation.